yellowbrick/text/postag.py

```diff
@@ -140,14 +141,16 @@ class PosTagVisualizer(TextVisualizer):
     """
 
     def __init__(
-        self,
-        ax=None,
-        tagset="penn_treebank",
-        colormap=None,
-        colors=None,
-        frequency=False,
-        stack=False,
-        **kwargs
+            self,
+            ax=None,
+            tagset="penn_treebank",
+            colormap=None,
+            colors=None,
+            frequency=False,
+            stack=False,
+            parser=None,
```

Comment:
Would you mind adding the argument documentation to the docstring under parameters?

Reply:
Sure, resolved in 666909c